### DISCUSSION TITLE
How to return parse data by scenario?

### DISCUSSION THREAD

**Discussion by paulocoutinhox** [nlohmann/json#2881] [Q&A]
Hi,

How to return parse data by scenario?

This is my example:

```
namespace ezored
{
namespace domain
{

NLOHMANN_DEFINE_TYPE_NON_INTRUSIVE(Todo, id, title, body, data, done, createdAt, updatedAt)

} // namespace domain
} // namespace ezored
```

On some APIs i want send the json only with "id", "title", "body", "data", "done".

But on others, i want send the json with "id", "title", "done".

There is any way to filter the fields by name when de-serialize?

```
nlohmann::json json = resp;
responseStream << json; // only with fields [id, title, done] or [id, title, body, data]
```

Thanks.

**Comment by nlohmann** [comment]
No, you would need to do this manually with the erase function.
